committee-steering/governance/working-group-formation-process.md

```diff
@@ -0,0 +1,87 @@
+# Kubernetes Working Group Formation Process - Proposal
+
+Draft 1.0 // Jaice Singer DuMars // June, 2018
+
+## Process Overview and Motivations
+As opposed to “Birds of a Feather” or discussion groups, Working Groups provide a more formal avenue for disparate
+groups to intellectually collaborate around a common problem, craft a balanced position, and disband. Because they
+represent the interests of multiple special interest groups, they are a vehicle for consensus building, not
+construction. By design, the code under discussion resides in either multiple parts of the project, or is in the
+process of being extracted to an external repository.
+
+Because a Working Group is an acknowledged part of the Kubernetes governance model, a process is necessary for
+formation and disbanding, as well as guidance on what authority is vested in the group by the steering committee.
+
+## Goals of the process
+
+- An easy-to-navigate process for those wishing to establish and eventually disband a new Working Group
+- Simple guidance and differentiation on where a Working Group makes sense, and does not
+- Clear understanding that no authority is vested in a Working Group
+- Ensure all future Working Groups conform with this process
+
+## Non-goals of the process
+
+- Documenting other governance bodies such as sub-projects or SIGs
+- Changing the status of existing Working Groups/SIGs/Sub-projects
+
+## Is it a Working Group? Yes, if...
+- It does not own any code
+- It has a clear goal measured through a specific deliverable or deliverables
+- It is temporary in nature and will be disbanded after reaching its stated goal(s)
+- It requires input from more than one SIG/sub-project to come up with an adequately-vetted deliverable with reasonable
```

Comment:
Why is this item here? This seems unnecessarily limiting on the community. If a group of people in the Kubernetes community want to form a working group, why do they need the blessing of a SIG? This seems to vest SIGs with powers that they were never intended to have.

Taken to the extreme, this gives the SIGs collectively the power to limit the development of new ideas, even if the community wants to work on those ideas. 

It doesn't seem to me that there is anything gained by blocking on SIG approval. And "adequately-vetted" feels like somehow working groups need to petition the powers that be for approval. I don't that is in the spirit of a working group.

Reply:
Deleted